tests/unit/recommenders/evaluation/test_python_evaluation.py

```diff
@@ -200,6 +201,45 @@ def test_python_exp_var(rating_true, rating_pred):
     assert exp_var(rating_true, rating_pred) == pytest.approx(-6.4466, TOL)
 
 
+def test_get_top_k_items(rating_true):
+    top_3_items_df = get_top_k_items(
+        dataframe=rating_true,
+        col_user=DEFAULT_USER_COL,
+        col_rating=DEFAULT_RATING_COL,
+        k=3,
+    )
+    top_3_user_true = pd.Series([1, 1, 1, 2, 2, 2, 3, 3, 3])
+    top_3_rating_true = pd.Series([5, 4, 3, 5, 5, 3, 5, 5, 5])
+    top_3_rank_true = pd.Series([1, 2, 3, 1, 2, 3, 1, 2, 3])
+    assert(top_3_items_df[DEFAULT_USER_COL].equals(top_3_user_true))
+    assert(top_3_items_df[DEFAULT_RATING_COL].equals(top_3_rating_true))
+    assert(top_3_items_df['rank'].equals(top_3_rank_true))
+    assert(top_3_items_df[DEFAULT_ITEM_COL][:3].equals(pd.Series([1, 2, 3])))
+    assert(set(top_3_items_df[DEFAULT_ITEM_COL][3:5]) == set([1, 4]))
+    assert(top_3_items_df[DEFAULT_ITEM_COL][5] in [5, 6])
+    assert(set(top_3_items_df[DEFAULT_ITEM_COL][6:]) == set([2, 5, 6]))
```

Comment:
User 3. All three items have a score of 5, so the ordering is not unique.

Reply:
@ChuyangKe And this comment?